osc-staging.py

```diff
@@ -155,12 +157,18 @@ def do_staging(self, subcmd, opts, *args):
         elif cmd == 'freeze':
             for prj in args[1:]:
                 FreezeCommand(api).perform(api.prj_from_letter(prj))
+        elif cmd == 'acheck':
+            # Is it safe to accept? Meaning: /totest contains what it should and is not dirty
+            version_openqa = api.load_file_content("%s:Staging" % api.project, "dashboard", "version_totest")
```

Comment:
I have the feeling that this will appear more times, maybe is a good idea to create a method in the StagingAPI or something else to capture the logic (even with error management) to fetch info from the dashboard